aten/src/ATen/native/group_norm.cpp

```diff
@@ -133,5 +133,29 @@ Tensor group_norm(
 DEFINE_DISPATCH(GroupNormKernel);
 DEFINE_DISPATCH(GroupNormBackwardKernel);
 
+std::tuple<at::Tensor, at::Tensor, at::Tensor> math_group_norm(
```

Comment:
For CR purposes it would be helpful to cite where this is coming from

Reply:
I wrote the pytorch/xla version based on the old pytorch composite implementation. So it should be fine here ;)